### PR TITLE
Restructure project to be more consistent and less cluttered

### PR DESCRIPTION
**Pull request by AlmightyYakob**
@waxlamp lmk if you think there's room for improvement, I think this addresses most of the issues but I could have missed something.

### CODE REVIEW

File: multinet/Pipfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ recommonmark = "==0.5.0"
 pre-commit = "*"
 
 [packages]
-multinet = {path = "./multinet",editable = true}
+multinet = {path = ".",editable = true}
```
[the remark this answers is not in the document]
Comment (AlmightyYakob) [reply multinet/Pipfile]: Oops, sure thing!

[the remark this answers is not in the document]
Comment (waxlamp) [reply multinet/Pipfile]: Done in f1e29abc3f792ccf0edd9573c6dab30ddb069995.

**Comment by waxlamp** [comment]
@AlmightyYakob, take a look at my changes and leave a review (if the interface will let you; otherwise just comment your thoughts here).

**Comment by AlmightyYakob** [comment]
@waxlamp I would encourage you to try this locally and see if I missed anything. So far testing on my environment I had no issues.

**Comment by waxlamp** [comment]
Looks great; I just made one small change in the Pipfile.

**Comment by AlmightyYakob** [comment]
:tada:

**Pull request merged**